sessions/recap-project-1/quiz-app/README.md

```diff
@@ -12,13 +12,23 @@ able to:
 later project. But make sure you include the answer texts and bookmark buttons in your app. (You
 might want to look up the "hidden" attribute.)
 
+## Overview
+
+In this project we will have 3 pages:
+
+- **Home page:** were we will be able to view **all** questions.
```

Comment:
Home Page: Here, you'll have access to view all questions available on the platform.